cypress/support/components/actionspage.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+ */
+
+/** Navigate to Actions Page */
+function NavigateTo() {
+  cy.get('a[href$="/actions"]')
```

Comment:
Is there a reason you put `NavigateTo` here?

The selector is not part of the actions page.  I would expect the app page to have a set of functions to visit each of it's page (entities, actions, trainDialogs, logDialogs, settings).